Assets/Scripts/Screeps3D/Effects/NukeMissileArchRenderer.cs

```diff
@@ -19,8 +19,7 @@ public class NukeMissileArchRenderer : MonoBehaviour
     public GameObject point1;
     public GameObject point2;
     public Transform point3;
-
-    public GameObject missile;
+    public float nukeHeightOffset = 1.64f;
```

Comment:
What is this magic number? :) and should it be relative to `Constants.ShardHeight`?